WordPress/src/main/java/org/wordpress/android/ui/jetpack/common/viewholders/JetpackViewHolder.kt

```diff
@@ -1,17 +1,15 @@
 package org.wordpress.android.ui.jetpack.common.viewholders
 
 import android.view.LayoutInflater
-import android.view.View
 import android.view.ViewGroup
-import androidx.annotation.LayoutRes
 import androidx.recyclerview.widget.RecyclerView
-import kotlinx.android.extensions.LayoutContainer
+import androidx.viewbinding.ViewBinding
 import org.wordpress.android.ui.jetpack.common.JetpackListItemState
 
-abstract class JetpackViewHolder(
-    @LayoutRes layout: Int,
-    parent: ViewGroup,
-    override val containerView: View = LayoutInflater.from(parent.context).inflate(layout, parent, false)
-) : RecyclerView.ViewHolder(containerView), LayoutContainer {
+abstract class JetpackViewHolder<T : ViewBinding>(
+    val parent: ViewGroup,
+    val inflateBinding: (LayoutInflater, ViewGroup, Boolean) -> T,
+    val binding: T = inflateBinding(LayoutInflater.from(parent.context), parent, false)
```

Comment:
Do you think it makes sense to adjust the visibility of these parameters? 

```suggestion
    parent: ViewGroup,
    inflateBinding: (LayoutInflater, ViewGroup, Boolean) -> T,
    protected val binding: T = inflateBinding(LayoutInflater.from(parent.context), parent, false)
```